ckanext/ogdchcommands/logic.py

```diff
@@ -157,6 +193,23 @@ def ogdch_cleanup_resources(context, data_dict):
         log.debug("{} resources have been deleted together with their "
                   "dependencies: resource_revision and resource_view"
                   .format(count))
+
+    if not dryrun and delete_filestore_files:
```

Comment:
It would be nice to have a dryrun action for deleting the files as well, outputting a list of filepaths that would be deleted. Wdyt?